utils/yaml-parser.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"errors"
+	"gopkg.in/yaml.v2"
+)
+
+type DockerComposeConfig struct {
```

Comment:
These x-* values don't always exist in docker-compose files - what are we using them for?

Reply:
Yeah on reflection, grabbing these kind of values from here isn't really useful (as its more for local dev config)